signatures/slot.yml

```diff
@@ -115,3 +118,26 @@ Slot:
         type: Object[]
         description: Successfully reset slot
     errors: {}
+  AdminSlotReset:
+    path: /slot/admin/reset
+    method: POST
+    title: Admin reset Slot availablity
+    description: reset a slot
+    params:
+      hospital:
+        type: UUID
+        description: hospital_id
+      disease:
+        type: UUID
+        description: disease_id
+      date:
+        type: Optional<Date>
+        description: date of time_slot to reset
```

Comment:
Could you add an example of the `date` field?

Reply:
Actually, we will soon remove this field once the cron job is up.